mod-orders-storage/examples/po_line_collection.sample

```diff
@@ -63,7 +63,8 @@
           },
           "materialType": "f7e72403-2a13-43a4-a069-aaabe6c9dea8",
           "trial": false,
-          "userLimit": 10
+          "userLimit": 10,
+          "resourceUrl": "http://sample.com/document.pdf"
```

Comment:
Please use sample which links to real pdf document
Something like:
"https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf" or
"http://africau.edu/images/default/sample.pdf"